lib/base/error-handler.js

```diff
@@ -103,6 +104,7 @@ const log = require('npmlog'),
             "NOT_SUPPORT_RUNNINGLIST" : "Not supported method to get running app information",
             "NOT_SUPPORT_JOURNALD" : "journald does not support the option",
             "NOT_SUPPORT_PMLOGD" : "pmlogd does not support the option",
+            "NOT_SUPPORT_XZ" : "Not Supported xz file extension",
```

Comment:
It seems that need more kind guide. Such as 
"Not supported .xz extension. Only  analyze works with ipk wihch is packaged by CLI "

Reply:
I've updated this.